lib/parsers/moov_parser.rb

```diff
@@ -6,19 +6,21 @@ class FormatParser::MOOVParser
   # we can reasonably call "file type" (something
   # usable as a filename extension)
   FTYP_MAP = {
-    'qt  ' => :mov,
-    'mp4 ' => :mp4,
+    'crx ' => :cr3,
     'm4a ' => :m4a,
+    'mp4 ' => :mp4,
+    'qt  ' => :mov,
   }
 
   # https://tools.ietf.org/html/rfc4337#section-2
   # There is also video/quicktime which we should be able to capture
   # here, but there is currently no detection for MOVs versus MP4s
+  CR3_MIME_TYPE = 'image/x-canon-cr3'
   MP4_AU_MIME_TYPE = 'audio/mp4'
   MP4_MIXED_MIME_TYPE = 'video/mp4'
 
   def likely_match?(filename)
-    filename =~ /\.(mov|m4a|ma4|mp4|aac|m4v)$/i
+    filename =~ /\.(aac|cr3|m4a|m4v|ma4|mov|mp4)$/i
```

Comment:
I know this is not a change you made but I am somewhat surprised to see aac here. It's not in FTYP_MAP above and there is an entirely separate aac parser IIRC.

Reply:
Agreed, I'll take it out.